src/main/java/controller/RequestController.java

```diff
@@ -0,0 +1,138 @@
+            }
+            if (httpRequest.isPost()) {
+                String body = (String) httpRequest.getBody();
+
+                HttpQueryParams queryParams = HttpQueryParams.parseParams(body);
+
+                httpResponse = handleUserCreate(queryParams);
+            }
+        }
+
+        if (path.equals("/")) {
+            httpResponse = handleRoot();
+        }
+
+        if (httpResponse == null) {
+            return;
+        }
+        httpResponse.respond(dos);
+    }
+
+    private static HttpResponse handleUserCreate(HttpQueryParams queryParams) {
+        String userId = queryParams.get("userId");
+        String password = queryParams.get("password");
+        String name = queryParams.get("name");
+        String email = queryParams.get("email");
+        User user = new User(userId, password, name, email);
+
+        DataBase.addUser(user);
+
+        return HttpResponse.redirect("/index.html");
+    }
+
+    public static ContentType extractExt(String path) {
+        String[] tokens = path.split(PATH_DELIMITER);
+        if (tokens.length < 1) {
+            return null;
+        }
+        String file = tokens[tokens.length - 1];
+        String[] fileNames = file.split(EXT_DELIMITER);
+        if (fileNames.length <= 1) {
+            return null;
+        }
+        return ContentType.of(fileNames[fileNames.length - 1]);
+    }
+
+    private static boolean isFile(String path) {
+        String[] paths = path.split(QUERY_DELIMITER);
+        ContentType mime = extractExt(paths[0]);
+        return mime != null;
+    }
+
+    private static boolean existFile(String path) {
+        // ?: 상대경로로 안돼서 절대경로로 수정했습니다.
+        Path absolutePath = Paths.get(path).toAbsolutePath();
+        File file = new File(absolutePath.toString());
+        return file.exists();
+    }
+
+    private static HttpResponse handleFile(HttpRequest httpRequest) throws IOException {
+        String path = httpRequest.getPath();
+        // 1. template (html)
+        if (existFile("src/main/resources/templates" + path)) {
+            return handleFileResponse("templates", httpRequest);
+        }
+
+        // 2. static (css...)
+        return handleFileResponse("static", httpRequest);
+    }
+
+    private static HttpResponse handleRoot() {
+        byte[] body = "Hello World".getBytes();
+        HttpHeaders headers = new HttpHeaders();
+        headers.put("Content-Type", "text/html;charset=utf-8");
+
+        return new HttpResponse(HttpStatus.OK, headers, body);
+    }
+
+    private static HttpResponse handleFileResponse(String parentFolder, HttpRequest request) throws IOException {
+        try {
+            String path = request.getPath();
+            String filePath = parentFolder + path;
+
+            byte[] body = FileIoUtils.loadFileFromClasspath(filePath);
+            HttpHeaders headers = new HttpHeaders();
+            String mime = parseMIME(filePath);
+            headers.put("Content-Type", mime + ";charset=utf-8");
```

Comment:
headers.`put` 메서드에 `ContentType`을 메서드 파라미터로 받도록하면, 아래처럼 더 깔끔하게 코드를 변경해볼 수 있겠네요. 🙂
header의 key나 value를 잘못 입력해서 버그가 발생하는 일은 훨씬 적어질 것 같네요. 😉


```suggestion
            ContentType contentType = ContentType.of(filePath);
            headers.put(contentType);
```